frontend/src/credit_transfers/CreditTransferAddContainer.js

```diff
@@ -40,6 +40,7 @@ class CreditTransferAddContainer extends Component {
       this.state = {
         fields: {
           comment: '',
+          agreementDate: '',
```

Comment:
Just referring to the previous comment, this could be dateOfWrittenAgreement

Reply:
done